sdk/cdn/Azure.ResourceManager.Cdn/api/Azure.ResourceManager.Cdn.net8.0.cs

```diff
@@ -1619,6 +1711,16 @@ protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer
         public static bool operator !=(Azure.ResourceManager.Cdn.Models.CertificateDeleteAction left, Azure.ResourceManager.Cdn.Models.CertificateDeleteAction right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public abstract partial class CertificateSourceParameters : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Cdn.Models.CertificateSourceParameters>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Cdn.Models.CertificateSourceParameters>
```

Comment:
sorry I did not notice this last time, this type should be renamed to `CertificateSourceProperties`.
A type ending with `Parameters` will trigger the linter error.